lib/logproto/tests/test-regexp-multiline-server.c

```diff
@@ -29,6 +29,11 @@
 #include "logproto/logproto-regexp-multiline-server.h"
 
 #include <criterion/criterion.h>
+#include <criterion/parameterized.h>
+
+typedef struct LogTransportMockConstructor {
+	LogTransportMockConstructor log_transport_mock_new;
+} LogTransportMockConstructorParam;
```

Comment:
This struct is not necessary, you can use `LogTransportMockConstructor` instead of `LogTransportMockConstructorParam`.

You've also redefined the name `LogTransportMockConstructor` (`struct LogTransportMockConstructor`), that is why Travis fails to compile the test:

https://travis-ci.org/balabit/syslog-ng/jobs/516846404#L2259